src/frontend/screens/Settings/sections/GamesSettings/index.tsx

```diff
@@ -47,18 +47,23 @@ type Props = {
 export default function GamesSettings({ useDetails = true }: Props) {
   const { t } = useTranslation()
   const { platform } = useContext(ContextProvider)
-  const { isDefault, gameInfo, isMacNative, isLinuxNative } =
-    useContext(SettingsContext)
+  const { isDefault, gameInfo, isLinuxNative } = useContext(SettingsContext)
   const [wineVersion] = useSetting('wineVersion', defaultWineVersion)
   const isLinux = platform === 'linux'
-  const isWin = platform === 'win32'
   const isCrossover = wineVersion?.type === 'crossover'
   const hasCloudSaves = gameInfo?.cloud_save_enabled
 
-  const nativeGame =
-    isWin ||
-    (isMacNative && gameInfo?.install.platform === 'Mac') ||
-    (isLinuxNative && gameInfo?.install.platform === 'linux')
+  const [nativeGame, checkNativeGame] = useState(false)
+
+  useEffect(() => {
```

Comment:
Ideally we should use a async function here because otherwise we will won't block the rendering and the value will change after the component is loading.
Also the pattern for the useState is not good. if the value is `nativeGame` the setter should be `setNativeGame`.

```
  useEffect(() => {
  const getIsNative = async () => {
  if (gameInfo){
   const isNative = await window.api...
   setNativeGame(isNative)
  }
  getIsNative()
  }
```

Reply:
Noted.

Reply:
Using async is causing the bug to occur again, i.e. displaying Wine options for Linux Sideloaded games.
```ts
useEffect(() => {
    const getIsNative = async () => {
      if (gameInfo) {
        const isNative = await window.api.isNative({
          appName: gameInfo?.app_name,
          runner: gameInfo?.runner
        })
        setNativeGame(isNative)
      }
      getIsNative()
    }
  })
```

Reply:
Just needed to move the `gameInfo` if check above and it works -
```ts
useEffect(() => {
    if (gameInfo) {
      const getIsNative = async () => {
       ............
    }
  }
```

Reply:
If the above looks good to you, I'll push this.